.azure/linux-stack.yml

```diff
@@ -73,10 +89,20 @@ jobs:
       source .azure/linux.bashrc
       stack install --resolver=lts-11.18 liquid-fixpoint-0.7.0.7 dotgen-0.4.2 fgl-visualize-0.1.0.1 located-base-0.1.1.1 liquidhaskell-0.8.2.4
     displayName: "Install Runtime Test-Dependencies: liquidhaskell"
+  - bash: |
+      source .azure/linux.bashrc
+      stack install hoogle --stack-yaml=$(YAML_FILE)
```

Comment:
`stack build hoogle` should be enough, iirc.

Reply:
but then `stack run` should be used in the next line, no?
I used install+exec taking the circleci config as a model

Reply:
I thought `stack run` doesnt exist aynmore? 
However, in install.hs we are using `stack build` and `stack exec`.

Reply:
sorry i wanted to mean poorly that i just have tested `stack build + stack exec` and it worked, so i will change